static/data/translation-table.json

```
@@ -728,7 +738,8 @@
     },
     {
       "cn": "鲜味",
-      "en": "umami taste",
+      "en": "umami", 
+      "note": "umami 就是一种味道，yellow 就是一种颜色，不需翻译为 yellow color",
```

```suggestion
      "en": "umami taste", 
```
umami 作为一个外来词，引入以后后面跟一个固有词做修饰是很常见的语言现象，即使这实际上存在语意重复
比如 shiitake mushroom, yakitori grill, Mississippi River甚至广州话里面的 “珠江河”